packages/db/src/migrations/20250915152638_create_granule_groups_table_and_index.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_groups', (table) => {
+    table
+      .increments('cumulus_id')
+      .primary();
+    table
+      .integer('granule_cumulus_id')
+      .references('cumulus_id')
+      .inTable('granules')
+      .notNullable()
+      .onDelete('CASCADE');
+    table
+      .specificType('status', 'char(1)')
+      .comment('Granule active status')
+      .notNullable();
+    table
+      .integer('group_id')
+      .comment('Granule duplicate group id')
+      .notNullable();
+    table
+      .timestamps(false, true);
+    table
+      .unique(['granule_cumulus_id']);
```

Comment:
A unique constraint automatically creates a unique index, so adding another one could result in two unique indexes on the same column with different names.

Reply:
Will granule_cumulus_id be unique? Can a granule associated with multiple groups and multiple active granules?
If it's unique, why don't we use it  as a primary key?